app/uk/gov/hmrc/gform/upscan/UpscanController.scala

```diff
@@ -112,7 +112,7 @@ class UpscanController(
                         "generic.error.virus",
                         formModelOptics.formModelVisibilityOptics.fcLookup
                           .get(formComponentId)
-                          .map(fc => fc.shortName.map(_.value).getOrElse(fc.label.value))
+                          .map(fc => (fc.errorShortName orElse fc.shortName).map(_.value).getOrElse(fc.label.value))
```

Comment:
Let's  not duplicate this `orElse` 5 times. We can put it as a new method on `FormComponent`

Reply:
done